README.md

```diff
@@ -28,7 +28,14 @@
 1. To run the following tests, run `npm test`.
     1. Unit testing
     2. Mock testing
-2. To run the ui tests, run `npm run test-ui`
+2. To run the ui tests:
+    - Copy '.env.example' 
+    - Rename the file to .env
+    - Login tests, run -  `npm run test-ui:login`
+    - View Job Roles tests, run -  `npm run test-ui:roles`
+    - To run local - change 'process.env.HOMEPAGE_URL' to 'process.env.HOMEPAGE_URL_LOCAL' for all lines in the file.
```

Comment:
Would it not be better to have another file with this already done, and make a new command to run it locally instead

Reply:
found another solution which was approved by mentors